sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -83,6 +153,21 @@ export interface PollOperationState<TResult> {
 // @public
 export type PollProgressCallback<TState> = (state: TState) => void;
 
+// @public
+export type RawHttpHeaders = {
+    [headerName: string]: string;
```

Comment:
is this collection case sensitive? I wonder if it will cause any trouble, since we did the work to make our headers maps case insensitive

Reply:
Good question I just call `toJSON()` or `rawHeaders()` from `core-client` and `core-http` respectively to populate this.

Reply:
I removed `RawHttpHeaders` and just inlined its definition since it is used only once.